projects/neoforge/build.gradle

```diff
@@ -10,17 +10,27 @@ plugins {
     id 'neoforge.versioning'
 }
 
+apply plugin : net.neoforged.neodev.NeoDevPlugin
```

Comment:
I don't love `apply plugin`, if we use the plugin development plugin in `buildSrc` and give the plugin an ID we can apply it to projects using the ID.

Reply:
No, but I think it's ugly :P Besides, giving the plugins IDs isn't very hard:
```groovy
gradlePlugin {
    plugins {
        create('plugin') {
            id = 'id'
            implementationClass = 'net.blah.Blah'
        }
    }
}
```

Reply:
I don't know. We'd have to make up plugin ids. Doesn't matter so I'm not gonna change it. 🤷

Reply:
`net.neoforged.neodev` and `net.neoforged.neodev.extra` 😛